packages/checks/src/is-hex-color-string.ts

```diff
@@ -0,0 +1,4 @@
+import { isString } from "util";
```

Comment:
This import is wrong, and should be:

```ts
import { isString } from "./is-string";
```

Btw. These wrong auto-imports are often biting me too in VSCode, especially with new source files. To remedy, I do the following:

- save the file
- choose "Restart TS server" from VS command pallette
- then add/quickfix missing imports

Reply:
👍